packages/codebytes/src/index.tsx

```diff
@@ -1,24 +1,79 @@
+import { Box, IconButton } from '@codecademy/gamut';
+import { FaviconIcon } from '@codecademy/gamut-icons';
+import { Background, states, system } from '@codecademy/gamut-styles';
+import { StyleProps } from '@codecademy/variance';
+import styled from '@emotion/styled';
 import React, { useState } from 'react';
 
+import { languageOption } from './consts';
 import { Editor } from './editor';
 
+const editorStates = states({
+  isIFrame: { height: '100vh' },
+});
+
+const editorBaseStyles = system.css({
+  border: 1,
+  borderColor: 'gray-900',
+  display: 'flex',
+  flexDirection: 'column',
+  height: '25rem',
+  width: '43rem',
+  overflow: 'hidden',
+});
+export interface EditorStyleProps
+  extends StyleProps<typeof editorBaseStyles>,
+    StyleProps<typeof editorStates> {}
+
+const EditorContainer = styled(Background)<EditorStyleProps>(
+  editorBaseStyles,
+  editorStates
+);
+
 export interface CodeByteEditorProps {
   text: string;
+  language: languageOption;
+  hideCopyButton: boolean;
+  onCopy?: (text: string, language: string) => void;
+  isIFrame?: boolean;
+  snippetsBaseUrl?: string;
+  onTextChange?: (text: string) => void;
 }
 
 export const CodeByteEditor: React.FC<CodeByteEditorProps> = ({
   text: initialText,
+  language,
+  hideCopyButton,
+  onCopy,
+  isIFrame = false,
+  snippetsBaseUrl = process.env.CONTAINER_API_BASE,
```

Comment:
can we just set this to an empty string with a TODO comment linking to the ticket that will handle? want to make sure we don't forget about it

Reply:
sounds perfectly reasonable!